### PR TITLE
add AmPm clock face

### PR DESCRIPTION
**Pull request by Maciej-Mudita**
**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

File: products/BellHybrid/apps/common/include/common/layouts/HomeScreenLayoutClassicWithQuotes.hpp [hunk 1]
```diff
@@ -12,16 +12,12 @@ namespace gui
       public:
         HomeScreenLayoutClassicWithQuotes(std::string name);
```
Comment (Lefucjusz) [inline products/BellHybrid/apps/common/include/common/layouts/HomeScreenLayoutClassicWithQuotes.hpp:13]: Consider marking this constructor `explicit`, as it takes only one parameter, also passing the name via `const &` should avoid one unnecessary copying of the string.

File: products/BellHybrid/apps/common/include/common/layouts/HomeScreenLayoutClassicWithQuotesAmPm.hpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#pragma once
+
+#include "HomeScreenLayoutClassicWithQuotes.hpp"
+
+namespace gui
+{
+    class HomeScreenLayoutClassicWithQuotesAmPm : public HomeScreenLayoutClassicWithQuotes
+    {
+      public:
+        HomeScreenLayoutClassicWithQuotesAmPm(std::string name);
```
Comment (Lefucjusz) [inline products/BellHybrid/apps/common/include/common/layouts/HomeScreenLayoutClassicWithQuotesAmPm.hpp:13]: The same suggestion as above :wink:

**Review by Lefucjusz — APPROVED** [review]

**Review by dawidwojtas-mudita — APPROVED** [review]

File: products/BellHybrid/apps/common/include/common/layouts/HomeScreenLayoutClassicWithQuotes.hpp (hunk 1 above)
Comment (Maciej-Mudita) [reply products/BellHybrid/apps/common/include/common/layouts/HomeScreenLayoutClassicWithQuotes.hpp]: done

File: products/BellHybrid/apps/common/include/common/layouts/HomeScreenLayoutClassicWithQuotesAmPm.hpp (hunk 2 above)
Comment (Maciej-Mudita) [reply products/BellHybrid/apps/common/include/common/layouts/HomeScreenLayoutClassicWithQuotesAmPm.hpp]: done

**Pull request merged**